src/core/Executor.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.23;
 import { Execution } from "../interfaces/IMSA.sol";
 
 /**
- * @title Execution
```

Comment:
good shout. we should probably change the file name instead, since Executor is the name of Executor Modules, and this might cause some irritation